openshift4/tusd/overlays/prod/patch.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tusd
+spec:
+  template:
+    spec:
+      containers:
+        - name: tusd
+          image: "image-registry.openshift-image-registry.svc:5000/4c2ba9-tools/tusd:prod"
+          env:
+            - name: S3_PREFIX
+              value: mds-files-prod/
```

Comment:
You read my mind already. Nice! You can disregard the prior comment. for specifying test/prod